package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@asyncapi/specs",
-  "version": "2.13.2",
+  "version": "2.14.0-2022-04-release.3",
```

Comment:
This should be probably changed to the `2.13.2`

Reply:
no, RC should stay, `semantic-release` will figure it out, notice 2.13 and bump to 2.14 🍻

Reply:
Great ❤️